src/NzbDrone.Core/NetImport/TMDb/TMDbSettings.cs

```diff
@@ -93,10 +101,64 @@ public TMDbSettings()
         [FieldDefinition(8, Label = "Original Language", Type = FieldType.Select, SelectOptions = typeof(TMDbLanguageCodes), HelpText = "Filter by Language")]
         public int LanguageCode { get; set; }
 
+        [FieldDefinition(9, Label = "Person ID", HelpText = "Used when a People* list is selected.  for the url https://www.themoviedb.org/person/6384-keanu-reeves, the person id is '6384'. Only one person per list ")]
```

Comment:
I think this will get really confusing with this many options. I would prefer having a tag field where a use can just enter what they want, something like here: https://github.com/Radarr/Radarr/blob/0ce58570949362589e3ac9d313c42e38d99c3e94/src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs#L50

Reply:
First rule of data systems, never reuse fields.  it comes back to bite you 10 fold every time.  

The problem isn't the settings, all forms of external interfaces are complex in their own way and need to have settings and parameters.  the problem is the stupid simple screen.  you should be able to change the settings that are displayed based on what list type you select.  but this UI can't do that.  I spent a lot of time thinking about it when planning this and the real fix is to rebuild that setting screen so that it can display the settings and parameters in a more intelligent way.  But im a backed data guy, i don't do UI development.  Figured in your branch where you are rebuilding the UI to Sonarr 3's UI, we can address it there

reusing fields is a bad idea, 100% of the time.  When I ran dev teams, that was a mistake that justified public scorn and abject humiliation during code reviews.